{{ cookiecutter.package_name|slugify }}/README.md

```diff
@@ -56,37 +62,29 @@ To add and install this package as a dependency of your project, run `poetry add
+1. [Add your private package repository credentials to your Poetry's `auth.toml` file](https://python-poetry.org/docs/repositories/#configuring-credentials):
+    ```toml
+    # Linux:   ~/.config/pypoetry/auth.toml
+    # macOS:   ~/Library/Application Support/pypoetry/auth.toml
+    # Windows: C:\Users\%USERNAME%\AppData\Roaming\pypoetry\auth.toml
+    [http-basic.{{ cookiecutter.private_package_repository_name|slugify }}]
+    username = "{username}"
+    password = "{password}"
+    ```
+1. Create a `.env` file in the project directory that [Docker Compose reads](https://docs.docker.com/compose/env-file/) to pass Poetry's `auth.toml` file as a [build and run time secret](https://docs.docker.com/compose/compose-file/compose-file-v3/#secrets-configuration-reference):
     ```sh
-    # bash
-    echo "export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_USERNAME='{username}'" >> ~/.bash_profile
-    echo "export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_PASSWORD='{password}'" >> ~/.bash_profile
-    
-    # fish
-    echo "set --export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_USERNAME '{username}'" >> ~/.config/fish/config.fish
-    echo "set --export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_PASSWORD '{password}'" >> ~/.config/fish/config.fish
-
-    # zsh
-    echo "export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_USERNAME='{username}'" >> ~/.zshenv
-    echo "export POETRY_HTTP_BASIC_{{ cookiecutter.private_package_repository_name|upper|replace("-", "_") }}_PASSWORD='{password}'" >> ~/.zshenv
+    # Linux
+    POETRY_AUTH_TOML_PATH=~/.config/pypoetry/auth.toml
+
+    # macOS
+    POETRY_AUTH_TOML_PATH=~/Library/Application Support/pypoetry/auth.toml
+
+    # Windows
+    POETRY_AUTH_TOML_PATH=$APPDATA/pypoetry/auth.toml
     ```
 {%- endif %}
 {%- endif %}
-1. [Install Docker Desktop](https://www.docker.com/get-started).
-1. [Configure Docker and Docker Compose to use the BuildKit build system](https://pythonspeed.com/articles/docker-buildkit/):
-    ```sh
-    # bash
-    echo "export DOCKER_BUILDKIT=1" >> ~/.bash_profile
-    echo "export COMPOSE_DOCKER_CLI_BUILD=1" >> ~/.bash_profile
-
-    # fish
-    echo "set --export DOCKER_BUILDKIT 1" >> ~/.config/fish/config.fish
-    echo "set --export COMPOSE_DOCKER_CLI_BUILD 1" >> ~/.config/fish/config.fish
-    
-    # zsh
-    echo "export DOCKER_BUILDKIT=1" >> ~/.zshenv
-    echo "export COMPOSE_DOCKER_CLI_BUILD=1" >> ~/.zshenv
-    ```
+1. [Install Docker Desktop](https://www.docker.com/get-started). If you are using Linux, you must [configure Docker and Docker Compose to use the BuildKit build system](https://pythonspeed.com/articles/docker-buildkit/). On macOS and Windows, BuildKit is enabled by default in Docker Desktop.
```

Comment:
From which version on is it default in Docker Desktop?

Reply:
From the article:
<img width="667" alt="image" src="https://user-images.githubusercontent.com/4543654/167405799-fa948032-2469-48ea-9abf-7735331c4f9f.png">